internal/extproc/watcher_test.go

```diff
@@ -54,7 +77,24 @@ func TestStartConfigWatcher(t *testing.T) {
 	path := tmpdir + "/config.yaml"
 	rcv := &mockReceiver{}
 
-	require.NoError(t, os.WriteFile(path, []byte{}, 0o600))
+	ctx, cancel := context.WithCancel(t.Context())
+	defer cancel()
```

Comment:
i think this cancel is unnecessary anymore so can you remove it 🙏

Reply:
Done!